index.ts

```diff
@@ -832,10 +826,6 @@ export class EthereumTuple extends Array<EthereumValue> {}
  * `Value` objects.
  */
 export class Entity extends TypedMap<string, Value> {
-  constructor() {
-    this.entries = new Array<TypedMapEntry<string, Value>>(0)
```

Comment:
Have you tested that this doesn't lead to the odd behavior we introduced this for?

Reply:
I didn't because that's hard to reproduce, but I did test that a base class with no constructor implicitly calls the super constructor.